docs/reference/announcements/870.md

```diff
@@ -0,0 +1,136 @@
+
+- Upcoming API Changes in Camunda 8
+- Introducing Enhanced Identity Management in Camunda 8.7
+- Streamlined Deployment with 8.7
+
+:::
+
+## API updates <span class="badge badge--long" title="This feature affects SaaS">SaaS</span><span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span>
+
+The 8.7 release includes API updates to support the move to a [Camunda 8 REST API](/apis-tools/camunda-api-rest/camunda-api-rest-overview.md) unified experience.
+
+### Camunda 8 REST API updates
+
+- New Query endpoints (with advanced search filtering) will be added for process entities (processes, decisions, user tasks, and forms). These will replace the component APIs (Tasklist, Operate) going forward.
+- New endpoints will allow you to manage and query users and resource permissions in an orchestration cluster.
+- All the Camunda 8 REST API endpoints will support resource-based authorizations to enable fine-grained permissions.
+- API terminology is aligned so technical assets have an identical, easily-understood, descriptive property name.
+
+### Deprecated: Operate and Tasklist REST APIs
+
+The deprecation process for the [Operate](/apis-tools/operate-api/overview.md) and [Tasklist](/apis-tools/tasklist-api-rest/tasklist-api-rest-overview.md) REST APIs starts with the 8.7 release. You can begin migrating to the [Camunda 8 REST API](/apis-tools/camunda-api-rest/camunda-api-rest-overview.md) for querying to prepare for this change.
+
+- Version 8.7, 8.8: These APIs are available, but not recommended for new implementations.
+- Version 8.9: These APIs will be deprecated and removed.
+
+### Deprecated: Job-based User Tasks querying
+
+As `Job-worker` user tasks will be deprecated in Camunda 8.9, Camunda recommends you start using `Camunda User Tasks` (formerly known as `Zeebe User Task`) in your process definitions.
+
+- Version 8.7, 8.8: `Job-worker` user tasks are available for querying, but Camunda modelers automatically apply the `Camunda user task` and show a warning message for each job worker user task.
+- Version 8.9: `Job-worker` user tasks will be deprecated. With Camunda 8.9 and later, customers can use the `Job-worker` implementation of user tasks as standard jobs with headers to enable open architecture and composable solutions.
+
+### Deprecated: Zeebe gRPC API endpoints
+
+Several [Zeebe gRPC](/apis-tools/zeebe-api/grpc.md) endpoints will be deprecated.
+
+- Key gRPC endpoints necessary for high-throughput and low-latency applications remain available with 8.7.
+- The final list of retained gRPC endpoints will be confirmed with the 8.7 release.
+- Selected endpoints will remain active, with others scheduled for removal in the 8.9 release.
+
+### Deprecated: Tasklist GraphQL API
+
+The [Tasklist GraphQL API](/apis-tools/tasklist-api/tasklist-api-overview.md) will be removed from the product once it has been fully deprecated.
```

Comment:
Can we publish the date of removal?

Reply:
@theburi We can, but do we even know this? If so, who might be best placed to answer this?

Reply:
@aleksander-dytko

Reply:
Related to https://github.com/camunda/camunda-docs/pull/4649

Reply:
@marcosgvieira can you help us here? Can we add a date or a version for removal?

Reply:
@mesellings see https://docs.google.com/document/d/1aqwSXP8uF3I7cV6Rz-HmBYIXSltwlmfjHKmmlVWBg6o/edit?tab=t.0#heading=h.zhyuwlliwpqj

We can announce this was (is?) removed in 8.7, but I'm still not sure if we can associate it with an alpha. For the date, we could set it for the minor release date at the latest (Feb 11).

Reply:
Thanks, I've amended the entry, I think we can close this conversation now 👍